### PR TITLE
add extra info when show index detail

### PR DESCRIPTION
**Pull request by morphes1995**
### What problem does this PR solve?

_Briefly describe what this PR aims to solve. Include background context that will help reviewers understand the purpose of the PR._

Issue link:#779

### Type of change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that could cause existing functionality not to work as expected)
- [ ] Documentation Update
- [ ] Refactoring
- [ ] Performance Improvement
- [ ] Test cases
- [ ] Python SDK impacted, Need to update PyPI
- [ ] Other (please describe):

### CODE REVIEW

**Review by JinHai-CN — APPROVED** [review]

**Comment by JinHai-CN** [comment]
@morphes1995 
thrift_server.cpp:1134, the result row count will be validated. This part is also need to update.

File: python/infinity/remote_thrift/infinity_thrift_rpc/infinity.thrift [hunk 1]
```diff
@@ -372,8 +372,12 @@ struct ShowIndexResponse {
...
 4: string table_name,
 5: string index_name,
-6: string store_dir,
-7: string index_info,
+6: string index_type,
+7: string index_column_names,
+8: string index_column_ids,
+9: string other_parameters,
+10: string store_dir,
+11: string index_info,
```
Comment (JinHai-CN) [inline python/infinity/remote_thrift/infinity_thrift_rpc/infinity.thrift:380]: We already has other_parameters field, index_info field can be removed.

**Review by JinHai-CN — CHANGES_REQUESTED** [review]
thrift IDL file need to update

**Comment by morphes1995** [comment]
> thrift IDL file need to update

ok ,i will remove index_info fields in ShowIndexResponse in infinity.thrift.  Should index_info in TableIndexInfo also be removed ?

**Comment by JinHai-CN** [comment]
> TableIndexInfo

Yes, please also remove it.

**Review by JinHai-CN — APPROVED** [review]

**Pull request merged**